project 2/styles.css

```diff
@@ -0,0 +1,154 @@
+* {
+    box-sizing: border-box;
+}
+
+h1 {
+    font-family: 'Big Shoulders Display', cursive;
+    font-weight: 900;
+    text-align: center;
+    font-size: 24.87pt;
+    letter-spacing: 1px;
+    margin-top: 40px;
+    color: #32302A;
+}
+
+
+ 
+ ul.info {
+    font-family: 'Big Shoulders Display', cursive;
+    font-weight: 800;
+    font-size: 12pt;
+    border-top: 4px solid #CCC1A9;
+    border-bottom: 4px solid #CCC1A9;
+    border-left: 4px solid #CCC1A9;
+    display:inline-block;
+    list-style: none;
+    padding-left: 0;
+    margin-top: 1px;
+    margin-right: 15px;
+}
+li.info   {
+    border-right: 4px solid #CCC1A9;
+    display: inline-block;
+    padding: 0px 23px;
+  }
+li.special  {
+    
+    display: inline-block;
+    padding: 0px 15px;
+    
+  }
+  
+li.special + li.info {
+    border-left: 4px solid #CCC1A9;
+    padding: 0.1px 15px;
+    
+  }
+
+.special {
+    font-weight: 500;
+    
+}
+img {
+    width: 124px;
+    height: 179px;
+    border-radius: 0.25in;
+    display: inline-block; 
+    position: relative;
+    left: -37%; 
+    bottom: -23px;
+    
+}
+/* image source: ­­­­https://www.xn--sommerkinorntgenplatz-rec.ch/index.php/events */
+.text{
+    display: inline-block;
+    position: relative;
```

Comment:
I think you can achieve this same layout with fewer specific position rules; this would make your code more adaptable, flexible. If you had the inclination, I would see if you could get this down to 1-2 positioning rules and instead use padding and margin more.